docs/concepts/what-is-a-signal.md

```diff
@@ -12,7 +12,7 @@ A Signal is an external asynchronous request to a [Workflow Execution](/concepts
 
```

Comment:
Let's put these how-to links here
- How to develop Signals (link to app dev guide)
- How to send a Signal using tctl (link to tctl)

(and remove the how-to link at the bottom of this page.